anomaly-detection/build.gradle

```diff
@@ -0,0 +1,109 @@
+        include dependency("org.apache.lucene:lucene-sandbox:6.3.0")
+        include dependency("org.apache.lucene:lucene-spatial:6.3.0")
+        include dependency("org.apache.lucene:lucene-spatial-extras:6.3.0")
+        include dependency("org.apache.lucene:lucene-spatial3d:6.3.0")
+        include dependency("org.apache.lucene:lucene-suggest:6.3.0")
+        include dependency("org.elasticsearch:securesm:1.1")
+        include dependency("net.sf.jopt-simple:jopt-simple:5.0.2")
+        include dependency("com.carrotsearch:hppc:0.7.1")
+        include dependency("joda-time:joda-time:2.9.5")
+        include dependency("org.yaml:snakeyaml:1.15")
+        include dependency("com.fasterxml.jackson.dataformat:jackson-dataformat-smile:2.8.1")
+        include dependency("com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.8.1")
+        include dependency("com.fasterxml.jackson.dataformat:jackson-dataformat-cbor:2.8.1")
+        include dependency("com.tdunning:t-digest:3.0")
+        include dependency("org.hdrhistogram:HdrHistogram:2.1.6")
+        include dependency("net.java.dev.jna:jna:4.2.2")
+        include dependency("org.elasticsearch.plugin:transport-netty3-client:5.1.2")
+        include dependency("org.elasticsearch.plugin:transport-netty4-client:5.1.2")
+        include dependency("io.netty:netty-buffer:4.1.6.Final")
+        include dependency("io.netty:netty-codec:4.1.6.Final")
+        include dependency("io.netty:netty-codec-http:4.1.6.Final")
+        include dependency("io.netty:netty-common:4.1.6.Final")
+        include dependency("io.netty:netty-handler:4.1.6.Final")
+        include dependency("io.netty:netty-resolver:4.1.6.Final")
+        include dependency("io.netty:netty-transport:4.1.6.Final")
+        include dependency("org.elasticsearch.plugin:reindex-client:5.1.2")
+        include dependency("org.elasticsearch.client:rest:5.1.2")
+        include dependency("org.apache.httpcomponents:httpclient:4.5.2")
+        include dependency("org.apache.httpcomponents:httpcore:4.4.5")
+        include dependency("org.apache.httpcomponents:httpasyncclient:4.1.2")
+        include dependency("org.apache.httpcomponents:httpcore-nio:4.4.5")
+        include dependency("org.elasticsearch.plugin:lang-mustache-client:5.1.2")
+        include dependency("com.github.spullara.mustache.java:compiler:0.9.3")
+        include dependency("org.elasticsearch.plugin:percolator-client:5.1.2")
+        include dependency("org.apache.logging.log4j:log4j-to-slf4j:2.7")
+        include dependency("org.apache.logging.log4j:log4j-api:2.7")
+        include dependency("ch.qos.logback:logback-classic:1.1.7")
+        include dependency("ch.qos.logback:logback-core:1.1.7")
+
+    }
+}
+
+distributions {
+    main {
+        baseName = archivesBaseName
+        contents {
+            into('lib') {
+                from shadowJar
+                from(project.configurations.shadow)
+            }
+            into('conf') {
```

Comment:
To simplify, please move `logback.xml` and `app.json` files into `anomaly-detection/src/main/dist/conf`.  No special build steps are then required.